src/styles/.blocks/variables.css

```diff
@@ -0,0 +1,15 @@
+:root {
```

Comment:
this file should be in existing folder ```blocks``` or in ```styles```, there is no need to create new one ```.blocks```